lasagne/layers/special.py

```diff
@@ -352,3 +354,94 @@ def _meshgrid(height, width):
     ones = T.ones_like(x_t_flat)
     grid = T.concatenate([x_t_flat, y_t_flat, ones], axis=0)
     return grid
+
+
+class ParametricRectifierLayer(Layer):
+    """
+    lasagne.layers.ParametricRectifierLayer(incoming,
+    alpha=init.Constant(0.25), axes=1, **kwargs)
+
+    A layer that applies parametric rectify nonlinearity to its input
+    following [1]_ (http://arxiv.org/abs/1502.01852)
+
+    Equation for the parametric rectifier linear unit:
+    :math:`\\varphi(y_i) = \\max(x_i,0) + \\alpha_i \\min(x_i,0)`
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape
+
+    alpha : Theano shared variable, numpy array or callable
+        An initializer for the alpha parameters. If a shared variable or numpy
+        array is provided, the shape must match the incoming shape, skipping
+        those axes the alpha's are shared over (see below for an example).
```

Comment:
It says "see below for an example", but I don't see an example.  Sorry if I'm missing it.